src/java/fr/paris/lutece/plugins/forms/web/entrytype/EntryTypeDefaultDataService.java

```diff
@@ -212,6 +211,25 @@ public List<String> responseToStrings( FormQuestionResponse formQuestionResponse
 
         return listResponseValue;
     }
+    
+    private String getResponseValueForExport( Entry entry, Response response)
+    {
+    	String strResponseValue = null;
+    	IEntryTypeService entryTypeService = EntryTypeServiceManager.getEntryTypeService(entry);
+    	if (entryTypeService instanceof AbstractEntryTypeGeolocation)
```

Comment:
Shouldn't you create a new IEntryDataService a map it to the correct entry type, so as to not bring knowledge of datatypes specificities into the default IEntryDataService  ?

Reply:
My logic here is the way tha data are filled in the export depends on service from genatt plugin.
It's normally located on the getResponseValueForExport() method from AbstractEntryTypeGeolocation class from genatt plugin.

Since i don't want to create compatibilty issues, i made it to only make changes on plugin forms and try to handle this case like a specific case.

Reply:
Ok, but there is already a `EntryTypeGeolocDisplayService` in plugin-forms. You could create a `EntryTypeGeolocDataService` with your custom code for the geoloc entry type